pages/linux/shiny-mirrors.md

```diff
@@ -0,0 +1,21 @@
+# shiny-mirrors
+
+> Generate a pacman mirror list for Manjaro Linux.
+> Every run of shiny-mirrors requires you to synchronize your database and update your system using `sudo pacman -Syyu`.
+> More information: <https://gitlab.com/Arisa_Snowbell/shiny-mirrors/-/blob/domina/shiny-mirrors/man/shiny-mirrors.md>.
+
+- Get the status of the current mirrors:
+
+`shiny-mirrors status`
+
+- Generate a mirror list using the default behavior:
+
+`sudo shiny-mirrors refresh`
+
+- Display the current settings file:
+
+`shiny-mirrors config show`
+
+- Switch to a different branch interactively:
```

Comment:
This may be confused with a Git branch if you don't know much about the repos. How about this:
```suggestion
- Switch to a different branch (stable, testing, unstable) interactively:
```

Reply:
This can be different if it is Arm or X86-64
also stable-staging is a thing
I would let it as it is